server/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+### Fixed
+
+- Pypika: `replace` doesn't replace substring value even if it is matching the value to replace.
```

Comment:
I was confused about this, so here's another suggestion. I'm not sure its better 😄 
```suggestion
- Pypika: `replace` doesn't do partial match anymore, as intended.
```